README.md

```diff
@@ -18,6 +18,19 @@ build is broken, but you don't notice it until later when you
 push. For this reason it's recommended to take care with this feature
 and always push after changing a whitelisted config value.
 
+## JDK version
+
+This buildpack does not install the JDK. Instead, it uses the default JDK
+on the Heroku Cedar stack, which is JDK 7. If you'd prefer to use JDK 8, you
+can suplement this buildpack with the jvm-common buildpack by running these
+commands:
+
+```
+$ heroku buildpacks:clear
```

Comment:
No, Heroku has native support for multiple buildpacks now. 
https://devcenter.heroku.com/articles/using-multiple-buildpacks-for-an-app